draft-ietf-quic-transport.md

```diff
@@ -3183,7 +3183,8 @@ containing that information is acknowledged.
 
 * The most recent set of acknowledgments are sent in ACK frames.  An ACK frame
   SHOULD contain all unacknowledged acknowledgments, as described in
-  {{sending-ack-frames}}.
+  {{sending-ack-frames}}.  Retransmitting an ACK frame can result in an
```

Comment:
This para is not great. That second sentence totally doesn't belong here. Suggested rewrite:

"ACK frames carry the most recent set of acknowledgements and the current Ack Delay estimate. Retransmitting an old ACK frame with stale Ack Delay can cause the peer to generate an inflated RTT sample."

Then in 13.2.1 (sending-ack-frames), add this as the first paragraph:
"ACK frames SHOULD acknowledge both ack-eliciting and non-ack-eliciting packets that have been received but not yet acknowledged."

Reply:
There is already text that says "When only non-ACK-eliciting packets need to be acknowledged, an endpoint MAY wait until an ACK-eliciting packet has been received to bundle an ACK frame with outgoing frames."  I'm not sure that's sufficient, but your suggestion adds a SHOULD, which makes this change no longer editorial.  The ACK generation text needs a full rewrite, and I'd rather tackle that suggestion as part of that PR.

Reply:
I worked part of your first suggestion in.

The latter suggestion is now present(from a different PR).